Add sounds to sportsball and basketball

balderdash opened the pull request:
We have different sets of sounds for traditional bounce, basketball, and sportsball, so I pushed the config from bounce.js and blocks.js into skins.js. Also added the playSound block to the toolbox in the later basketball/sportsball levels.

Also, the hand selector is broken in sportsball right now because of missing hand asset, copied that from basketball.

Hamms commented:
~~This PR should be opened against `staging-next`~~

As established offline, this is intended to go into staging for HoC

Hamms commented on apps/src/bounce/skins.js:
```diff
@@ -238,34 +338,14 @@ exports.load = function (assetUrl, id) {
+  skin.customSounds = config.customSounds || {};
+  for (var sound in skin.customSounds) {
+    skin.customSounds[sound].urls =
+      skin.customSounds[sound].filenames.map(skin.assetUrl);
```
dang, that's pretty

balderdash commented:
Two more basketball/sportsball changes in this PR:
* Added sharing thumbnails for basketball and sportsball that aren't just a screenshot of bounce
* Added a third hand color

balderdash commented:
(Waiting until Tuesday's HOC triage to merge)
